src/components/Header/HeaderLink.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import styled from 'styled-components'
+import { NavLink, NavLinkProps } from 'react-router-dom'
+import transparentize from 'polished/lib/color/transparentize'
+
+import { ExternalLink } from '../../theme/components'
+
+const StyledNavLink = styled(NavLink)`
+  ${({ theme }) => theme.flexRowNoWrap}
+  align-items: left;
+  border-radius: 3rem;
+  outline: none;
+  cursor: pointer;
+  text-decoration: none;
+  color: ${({ theme }) => theme.text5};
+  width: fit-content;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19.5px;
+  font-family: 'Montserrat';
+  position: relative;
+
+  &.active {
+    font-weight: 600;
+    color: ${({ theme }) => theme.white};
+  }
+`
+
+const StyledExternalLink = styled(ExternalLink)`
+  ${({ theme }) => theme.flexRowNoWrap}
+  align-items: left;
+  outline: none;
+  cursor: ${({ disabled }) => (disabled ? 'not-allowed' : 'pointer')};
+  text-decoration: none;
+  color: ${({ theme, disabled }) => (disabled ? transparentize(0.6, theme.text5) : theme.text5)};
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19.5px;
+  width: fit-content;
+  text-decoration: none !important;
+  font-family: 'Montserrat';
```

Comment:
Same here. Do we have to declare the font-family, don't we inherit from global styles?

Reply:
Issue created 💪 

https://github.com/levelkdev/swapr-dapp/issues/948#issue-1226479243